README.md

```diff
@@ -157,76 +229,27 @@ Rosetta requires a few Celo Core Contracts
 
 * The list of required contracts is defined on `scripts/gen-contracts.go` file
 * Generation requires acces to `celo-blockchain` & `celo-monorepo`.
-* Generation assumes both projects are **already properly built**
+* Generation assumes both projects are **already properly built** (see above under instructions for running `rosetta` from source for more details on how to do this)
 * To run generator do `make gen-contracts`
 
-## How to build Docker Image
-
-Commands:
-  * `make docker-build`
-  * `make docker-publish`
-
 ## How to run rosetta-cli-checks
+
 Install the [`rosetta-cli`](https://github.com/coinbase/rosetta-cli) according to the instructions. Current testing has been done with `v0.5.16`.
-- Run the Rosetta service in the background for the respective network (currently only alfajores)
-- Run the CLI checks as follows:
-```
+
+* Run the Rosetta service in the background for the respective network (currently only alfajores)
+* Run the CLI checks as follows:
+
+```sh
 # alfajores; specify construction or data
 rosetta-cli check:construction --configuration-file PATH/TO/rosetta/rosetta-cli-conf/testnet/cli-config.json
```

Comment:
I had to run `./bin/rosetta-cli check:construction ...` to get this to run (from the directory I installed it), so recommending installing `rosetta-cli` into `/usr/local/bin` instead of where the command is run would be better. as well as adding instructions to update the command to point to the right path for the `cli-config.json` file

* Install [`rosetta-cli`](https://github.com/coinbase/rosetta-cli)

If on Mac, it is recommended to install `rosetta-cli` into `/usr/local/bin`
```sh
curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/master/scripts/install.sh | sh -s -- -b /usr/local/bin
```

Reply:
I'm hesitant to add the actual install instructions for the cli here since that'll be an additional piece that we need to ensure is kept up to date (with the `rosetta-cli` README), so I'd prefer deferring to their installation instructions for that reason. That said, I'll add a note that installing to `/usr/local/bin` will allow you to use `rosetta-cli` directly on the command line (without the path to it)